lib/components/user/account-page.js

```diff
@@ -63,7 +63,10 @@ class AccountPage extends Component {
 const mapStateToProps = (state, ownProps) => {
   const currentPath = state.router.location.pathname
   return {
-    loggedInUser: state.user.loggedInUser
+    isTermsOrVerifyPage:
+      currentPath === CREATE_ACCOUNT_TERMS_PATH || currentPath === CREATE_ACCOUNT_VERIFY_PATH,
+    loggedInUser: state.user.loggedInUser,
+    trips: state.user.loggedInUserMonitoredTrips
```

Comment:
Why are trips needed here?

Reply:
It's leftover orphan code, removed in https://github.com/opentripplanner/otp-react-redux/pull/323/commits/814d9df6102f1c34cf1fedb8c45ba57bb3f24233.